depfinder/inspection.py

```diff
@@ -47,7 +48,10 @@
 logger = logging.getLogger('depfinder')
 
 pyver = '%s.%s' % (sys.version_info.major, sys.version_info.minor)
-builtin_modules = stdlib_list(pyver)
+if version.parse(pyver) < version.parse("3.10"):
```

Comment:
```suggestion
if sys.version_info < (3, 10):
```
Wouldn't this be fine? And then `packaging` not needed.

Reply:
This is still a draft and pretty much in flux, don't worry about it now. BTW,  I already removed that b/c I refactor that part.

PS: I'd love a review when I finished it though :grimacing:

Reply:
Ok!